fontbe/src/glyphs.rs

```diff
@@ -123,33 +127,100 @@ impl Work<Context, Error> for GlyphWork {
         // Hopefully in time https://github.com/harfbuzz/boring-expansion-spec means we can drop this
         let glyph = cubics_to_quadratics(glyph);
 
-        // TODO refine (submodel) var model if glyph locations is a subset of var model locations
-
         match glyph {
             CheckedGlyph::Composite { name, components } => {
                 let composite = create_composite(context, &name, default_location, &components)?;
                 context.set_glyph(name, composite.into());
             }
-            CheckedGlyph::Contour { name, contours } => {
+            CheckedGlyph::Contour { name, paths } => {
                 // Draw the default outline of our simple glyph
-                let Some(path) = contours.get(default_location) else {
+                let Some(path) = paths.get(default_location) else {
                     return Err(Error::GlyphError(ir_glyph.name.clone(), GlyphProblem::MissingDefault));
                 };
                 let base_glyph = SimpleGlyph::from_kurbo(path).map_err(|e| Error::KurboError {
                     glyph_name: self.glyph_name.clone(),
                     kurbo_problem: e,
                     context: path.to_svg(),
                 })?;
-                context.set_glyph(name, base_glyph.into());
+                context.set_glyph(name.clone(), base_glyph.into());
+
+                let point_seqs: HashMap<_, _> = paths
+                    .into_iter()
+                    .map(|(loc, bez)| (loc, points(bez)))
```

Comment:
since we are already converting BezPath segments to points inside `SimpleGlyph::from_kurbo` immediately before this, it would make sense to do that once (inside write_fonts::tables::glyf) and reuse the same list of points for the gvar deltas.

By the way, I think you are missing the additional four phantom points at the end.
cf. fonttools' `_getCoordinatesAndControls` method of glyf table, used by varLib when building gvar: https://github.com/fonttools/fonttools/blob/3b9a73ff8379ab49d3ce35aaaaf04b3a7d9d1655/Lib/fontTools/ttLib/tables/_g_l_y_f.py#L369

Reply:
Switched to convert everything to SimpleGlyph then compute deltas. Haven't had time to look at 

I can now write a gvar that FontTools can't understand.

> missing the additional four phantom points at the end

Confirmed, not implemented yet :)

Reply:
Font now works. Punting some of the work to https://github.com/googlefonts/fontmake-rs/issues/240.